src/xo-app/pool/tab-logs.js

```diff
@@ -1,19 +1,55 @@
 import _ from 'intl'
-import ActionRow from 'action-row-button'
 import React, { Component } from 'react'
-import TabButton from 'tab-button'
+import SortedTable from 'sorted-table'
 import { deleteMessage } from 'xo'
 import { createPager, createSelector } from 'selectors'
 import { FormattedRelative, FormattedTime } from 'react-intl'
-import { Container, Row, Col } from 'grid'
 import {
   ceil,
-  isEmpty,
   map
 } from 'lodash'
 
 const LOGS_PER_PAGE = 10
 
+const COLUMNS = [
+  {
+    itemRenderer: log =>
+      <div>
```

Comment:
You could put this one on the previous line (and indent accordingly) to make the code clearer.